src/woo_publications/metadata/organisation_sync.py

```diff
@@ -0,0 +1,99 @@
+from io import StringIO
+from pathlib import Path
+
+from django.core.management import call_command
+
+import requests
+from glom import PathAccessError, T, glom
+
+from woo_publications.metadata.constants import OrganisationOrigins
+from woo_publications.metadata.models import Organisation
+
+MUNICIPALITY_WAARDENLIJST_URL = "https://repository.officiele-overheidspublicaties.nl/waardelijsten/rwc_gemeenten_compleet/4/json/rwc_gemeenten_compleet_4.json"
+SO_WAARDENLIJST_URL = "https://repository.officiele-overheidspublicaties.nl/waardelijsten/rwc_samenwerkingsorganisaties_compleet/1/json/rwc_samenwerkingsorganisaties_compleet_1.json"
+OORG_WAARDENLIJST_URL = "https://repository.officiele-overheidspublicaties.nl/waardelijsten/rwc_overige_overheidsorganisaties_compleet/8/json/rwc_overige_overheidsorganisaties_compleet_8.json"
+
+MUNICIPALITY_WAARDENLIJST_TYPE = "https://identifier.overheid.nl/tooi/def/ont/Gemeente"
+SO_WAARDENLIJST_TYPE = (
+    "https://identifier.overheid.nl/tooi/def/ont/Samenwerkingsorganisatie"
+)
+OORG_WAARDENLIJST_TYPE = (
+    "https://identifier.overheid.nl/tooi/def/ont/Overheidsorganisatie"
+)
+WAARDENLIJST_URLS = [
+    MUNICIPALITY_WAARDENLIJST_URL,
+    SO_WAARDENLIJST_URL,
+    OORG_WAARDENLIJST_URL,
+]
+
+TYPE_MAPPING = [
+    (
+        MUNICIPALITY_WAARDENLIJST_URL,
+        MUNICIPALITY_WAARDENLIJST_TYPE,
+        OrganisationOrigins.municipality_list,
+    ),
+    (SO_WAARDENLIJST_URL, SO_WAARDENLIJST_TYPE, OrganisationOrigins.so_list),
+    (OORG_WAARDENLIJST_URL, OORG_WAARDENLIJST_TYPE, OrganisationOrigins.oorg_list),
+]
+
+SPEC = {
+    "identifier": "@id",
+    "naam": T["http://www.w3.org/2000/01/rdf-schema#label"][0]["@value"],
+}
+
+
+class OrganisatieWaardenlijstError(Exception):
+    def __init__(self, message: str):
+        self.message = message
+        super().__init__(message)
+
+
+def update_organisation(file_path: Path):
+    for waardenlijst_url, waardenlijst_type, oorsprong in TYPE_MAPPING:
+        try:
+            response = requests.get(waardenlijst_url)
+        except requests.RequestException as err:
+            raise OrganisatieWaardenlijstError(
```

Comment:
Currently I raise all these errrors individually, but now the case is that if one url raises an error the next one in line won't trigger. Do we want to prevent this by keeping track of all the errors and collect them in the loop and raises them all at the same time at the end?

Reply:
Discussed in person - loop behaviour is fine, it's all-or-nothing for this management command :)